app/wizards/claims/invalid_provider_wizard.rb

```diff
@@ -13,6 +13,9 @@ def initialize(school:, claim:, created_by:, params:, state:, current_step: nil)
 
     def define_steps
       add_step(AddClaimWizard::ProviderStep)
+      return if mentors_with_claimable_hours.any?
+
+      add_step(UnableToAssignProviderStep)
```

Comment:
This is just a style preference on my behalf - 

`add_step(UnableToAssignProviderStep) unless mentors_with_claimable_hours.any?`